cinema/models.py

```diff
@@ -125,22 +166,94 @@ class Meta:
         ordering = ["name"]
 
 
+class CinemaPersonManager(models.Manager):
+    """
+    Custom Manager, adding extra manager methods.
+
+    Custom Manager used in 'CinemaPerson' model by extending base Manager class
+    and instantiating custom Manager in 'CinemaPerson' model.
+    """
+    def get_queryset(self):
+        """
+        Return QuerySet object with cached data of some related models.
+        """
+        return super().get_queryset().select_related("user", "country")
+
+    def get_brief_persons_data(self):
+        """
+        Return QuerySet object with data of specified fields of
+        'CinemaPerson' and 'User' models.
+        """
+        return super().get_queryset().select_related("user").values_list(
+            "pk", "user__first_name",  "user__last_name", named=True
+        )
+
+    def filter_by_search_word(self, search_word):
+        """
+        Return QuerySet object, value of selected field corresponds to
+        search word entered by visitor.
+        """
+        split_query = search_word.split()
```

Comment:
Why do you split `search_word`?
`split_query` Is bad name cause search_word.split() can not return `split_query`.

Reply:
ok